scripts/apps/monitoring/services/CardsService.ts

```diff
@@ -65,6 +65,8 @@ export function CardsService(search, session, desks, $location) {
 
         if (card.type === 'spike' || card.type === 'spike-personal') {
             params.spike = 'only';
+        } else if (card.type === 'personal' && card.sent) {
+            params.spike = 'include';
```

Comment:
I'd call it `include-sent` to be more precise

Reply:
this param just controls if spiked items will be included or excluded, it doesn't handle sent filtering

Reply:
it's good to have clear names even for local variables. In this case it's not clear what will be included when that parameter is set.

Reply:
not sure if I follow here, if you set `spike` to `include` it includes spiked items. if it's `spike = 'include-spiked';` that's just duplication, isn't it?

Reply:
You're right. It'd be helpful though to document possible values to `spike` option and their meanings.